src/Illuminate/Foundation/Console/MailMakeCommand.php

```diff
@@ -68,6 +73,30 @@ protected function writeMarkdownTemplate()
         $this->files->put($path, file_get_contents(__DIR__.'/stubs/markdown.stub'));
     }
 
+    /**
+     * Write the Blade template for the mailable.
+     *
+     * @return void
+     */
+    protected function writeView()
+    {
+        $path = $this->viewPath(
+            str_replace('.', '/', $this->getView()) . '.blade.php'
+        );
+
+        if (!$this->files->isDirectory(dirname($path))) {
+            $this->files->makeDirectory(dirname($path), 0755, true);
+        }
+
+        $stub = str_replace(
+            '{{ quote }}',
+            Inspiring::quotes()->random(),
```

Comment:
Can probably remove this since you're not actually using `{{ quote }}` anywhere in the stubs, not to mention an inspiring quote being injected into the mailable template makes no sense.

Reply:
View stub has a quote.

https://github.com/laravel/framework/blob/11.x/src/Illuminate/Foundation/Console/stubs/view.stub

Reply:
@ahinkle ah nice I for some reason didn't realize it was the view stub being used, I thought it was one of the ones in the PR. Thanks!